sdk/communication/communication-chat/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-chat",
-  "version": "1.0.0-beta.6",
+  "version": "1.0.0",
```

Comment:
There are more changes that need to be made for each package.

`package-version`: This is either in package.json or in the README.md for swagger. Here it's on line 11.
Then we should re-gen with autorest. 

@LuChen-Microsoft @juancamilor Did we lock the swagger url for Chat to point to the spec repo? We can't point to the int-gateway hosted file anymore.

`SDK_Version` constant:
There's a version in contants.ts for each package that needs updating to 1.0.0.

Reply:
@DominikMe the PR for the specs is going through. Will update all sdks when is approved (hopefully today/tomorrow)

Reply:
@juancamilor Can we update the README to point to the latest commit in the PR of the specs repo?
A moving target for the swagger has been a common problem with Chat because we who are outside of the Chat team can't confidently re-generate the code.
We've tried it before to just update the packageVersion for the user agent but the re-gen'd code would have a lot of changes as work has happened in the gateway. That's why we pin swagger commits.

Reply:
@DominikMe  The change for swagger url is already merged into master.
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/communication/communication-chat/swagger/README.md